include/mimick/mock.h

```diff
@@ -46,17 +46,12 @@ struct mmk_params {
     struct mmk_params *next;
 };
 
-#if defined(__GNUC__) && !defined(__clang__)
-// Force bit-field alignment to circumvent a bug in GCC
-// versions 6, 7, and 8, and a fixup notice on GCC 9.
-// See https://gcc.gnu.org/bugzilla/show_bug.cgi?id=88469.
-// For 32 bit machines in which bit-field alignment matters,
-// this will ensure correct alignment. For 64 bit machines
-// in which bit-field alignment matters, we can reasonably
-// expect the compiler to tighten up alignment.
-struct __attribute__((aligned(4))) mmk_mock_options {
-#else
 struct mmk_mock_options {
+#if defined(__GNUC__) && !defined(__clang__)
+    // Force bit-field alignment to circumvent a bug in GCC
+    // versions 6, 7, and 8, and a fixup notice on GCC 9.
+    // See https://gcc.gnu.org/bugzilla/show_bug.cgi?id=88469.
+    unsigned force_alignment__;
 #endif
```

Comment:
Hm, but doesn't this make the size of the struct 8 bytes (2 * sizeof(unsigned)) now?  Is that what you were going for?

Reply:
I was trying not to make it bigger than it needs to be, but the actual size of it is of no consequence. We just need another field to dominate the structure alignment. 

Though going through the reported bug, there may be a better solution. Hang on.

Reply:
See https://github.com/ros2/Mimick/pull/11/commits/26ebdea4d6c1649684183a354516749dde314129. It's a bit more convoluted but not that much.